### PR TITLE
AP_Mount: Change mount stabilize limit and typos

### PR DESCRIPTION
**Pull request by Naterater**
After discussion with Tridge we found that the hard limit of the maximum mount stabilization angle was pi/3 radians or 60 degrees. We decided to increase the maximum stabilization angle from 60 degrees to 80 degrees for higher bank angles.

This is my first commit and pull request from my desktop. Please review thoroughly.

### CODE REVIEW

File: libraries/AP_Mount/AP_Mount_Servo.cpp [hunk 1]
```diff
@@ -169,11 +169,11 @@ void AP_Mount_Servo::stabilize()
 // closest_limit - returns closest angle to 'angle' taking into account limits.  all angles are in degrees * 10
 int16_t AP_Mount_Servo::closest_limit(int16_t angle, int16_t angle_min, int16_t angle_max)
 {
-    // Make sure the angle lies in the interval [-180 .. 180[ degrees
```
Comment (amilcarlucas) [inline libraries/AP_Mount/AP_Mount_Servo.cpp]: The [ means that 180 is not inclusive. So please revert this changes, the code was already OK

Comment (magicrub) [reply libraries/AP_Mount/AP_Mount_Servo.cpp]: If that's the case, id rather see a more universal sign such as < vs <= to be absolutely clear.

Comment (Naterater) [reply libraries/AP_Mount/AP_Mount_Servo.cpp]: I personally learned that [] was inclusive whereas () was not inclusive. So [-180,180) meant the range of -180 inclusive to 180 not inclusive. It sounds like everybody is familiar with different notations.

Comment (amilcarlucas) [reply libraries/AP_Mount/AP_Mount_Servo.cpp]: yes, sadly it looks like there are different math notations around the world :(

Comment (OXINARF) [reply libraries/AP_Mount/AP_Mount_Servo.cpp]: Insane is the word that occurs to me. It's the first time I hear about this other notation. According to https://en.wikipedia.org/wiki/ISO_31-11, *(a, b)* means both an open interval and an ordered pair; smart...

If these comments aren't universal we should change them, but the current change is wrong. Actually these `while`s should go away and `wrap_180_cd` used instead.
